id: PYSEC-0000-CVE-2014-2573
details: The VMWare driver in OpenStack Compute (Nova) 2013.2 through 2013.2.2 does
  not properly put VMs into RESCUE status, which allows remote authenticated users
  to bypass the quota limit and cause a denial of service (resource consumption) by
  requesting the VM be put into rescue and then deleting the image.
affected:
- package:
    name: nova
    ecosystem: PyPI
    purl: pkg:pypi/nova
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: "0"
references:
- type: WEB
  url: http://www.openwall.com/lists/oss-security/2014/03/21/1
- type: WEB
  url: http://www.openwall.com/lists/oss-security/2014/03/21/2
- type: WEB
  url: https://bugs.launchpad.net/nova/+bug/1269418
- type: ADVISORY
  url: http://secunia.com/advisories/57498
aliases:
- CVE-2014-2573
modified: "2014-03-26T13:41:00Z"
published: "2014-03-25T16:55:00Z"
